src/pages/react/training/react-fundamentals/london/index.js

```diff
@@ -129,17 +131,20 @@ const ReactFundamentalsLondon = () => (
           <Section>
             <Grid>
               <Row>
-                <HideComponentsUsingCss xs sm>
-                  <Col md={6} lg={5}>
-                    <Image src={BOOTCAMP_COLLAB} width="100%" />
-                  </Col>
-                </HideComponentsUsingCss>
-                <Col md={6} lg={5} lgOffset={1}>
+                <Col md={5} mdOffset={1}>
+                  <AttendeeQuote
+                    small
+                    quote="Coaches to student ratio was amazing - basically 1 coach for every 2 students. Having that helped accelerate my learning"
```

Comment:
@shakersinthedark that case, where there was 1 coach for every 2 students, was an exception because we didn't sell many tickets in that bootcamp. It should not be the norm. I'm not sure we should say that because it'll set the wrong expectations.

Reply:
OK, it's a real badass USP though so I'll keep the video but change that text? Then at least, it's not being shouted from the rooftops and users have to click into the video to discover that information.... I think that's a fair no?

Reply:
@alexlbr OK, i'll replace this video completely after our conversation :)